### PR TITLE
Show Labels For FAB in Contribution Page

### LINKED ISSUE #5470: Show Details About Icon Labels when long pressed
### What is the user problem or growth opportunity you want to see solved?

In the Nearby Page when we click on a Place we see multiple Icons in the bottom sheet like Wikipedia and Bookmark but to a new user it not be clear which Icon means what and it's quite confusing. In general it's a good practice in Android Development to show more context for icons and buttons.

### How do you know that this problem exists today? Why is this important?

I want to Implement a feature that when these Icons are Pressed for long then a Toast should appear Indicating the label of the Icon giving user Information about what clicking on Icon will do

### Who will benefit from it?

Users will benefit from this as it will Improve User Experience

### Anything else you would like to add?

I have implemented this on one of the Icons to showcase the Impact of this

https://github.com/commons-app/apps-android-commons/assets/126143257/5e6af6f9-8d39-41f5-8dbb-77d3463f2b55

I would like to add context like this on Icons in the app which might be confusing to users.

### PR DESCRIPTION
**Pull request by shashankiitbhu**
**Description (required)**

Fixes #5470 

What changes did you make and why?
Show a Toast Message that gives context about the working of button when Long Pressed
**Tests performed (required)**

Tested 4.2.1-debug on Xiaomi 11 Lite NE with API level 33

**Screenshots (for UI changes only)**
WhatsApp Image 2024-01-29 at 4 47 00 PM

### CODE REVIEW

**Comment by shashankiitbhu** [comment]
Please Review this @nicolas-raoul

File: app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java [hunk 1]
```diff
@@ -321,10 +325,18 @@ private void setListeners() {
             controller.initiateCameraPick(getActivity(), inAppCameraLocationPermissionLauncher);
             animateFAB(isFabOpen);
         });
+        binding.fabCamera.setOnLongClickListener(view -> {
+            ViewUtil.showShortToast(getContext(),R.string.add_contribution_from_camera);
```
Comment (domdomegg) [inline app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java:329]: I'm not sure this is true - this isn't just to add contributions from the camera (e.g. you can add it from the gallery).

Comment (shashankiitbhu) [reply app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java:329]: This is for a fab camera, one you can see below


The first one of the three, doesn't  the first one only opens camera ?




On my device this opens, there is not a way to open gallery from here.

Comment (domdomegg) [reply app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java:329]: Apologies my bad, you're right!

**Review by domdomegg — APPROVED** [review]

**Pull request merged**